packages/sdk-auth/src/tokenProvider.js

```diff
@@ -79,6 +79,9 @@ export default class TokenProvider {
     return this.fetchTokenInfoPromise.then((tokenInfo) => {
       this.fetchTokenInfoPromise = null
       return tokenInfo
+    }).catch((error) => {
+        this.fetchTokenInfoPromise = null
+        throw new Error(error)
```

Comment:
Why do we wrap the error inside a new error? Wouldnt it make more sense to just rethrow it?

`throw error`

Reply:
it is not error  but error message so we can not throw only string.

Reply:
Done

Reply:
huh?

```
const a = () => {throw new Error('failing')}
const b = () => {try { a();} catch (error) {console.log(error instanceof Error)}; }

b()
// true
```

Reply:
@daern91 didn't realize that check above comment I already did ;-)